### ISSUE TITLE
Can't run batches more than 10 after latest commits

### ISSUE THREAD

**Issue by JamesIV4** [hlky/stable-diffusion#218]
I like to run very large batches while I step away. Now I can only go up to 10. Please, consider adding that flexibility back.

The new update is awesome btw, so many cool new features.

**Comment by Aishou** [comment]
Second that, it's kinda limiting after all for no apparent reason.... 
In the meantime, you can just change it here ;)
Please give back some flexibility, it's nice to be able to just step back and wonder what's coming out of that batch :D
https://github.com/hlky/stable-diffusion/blob/e2897e6b17579b499578d2e88f6337025adaae08/frontend/frontend.py#L34

**Comment by JamesIV4** [comment]
Thanks! That's super helpful.

**Comment by hlky** [comment]
I agree @altryne lowered it to 10 and it's a bit low. It was 250 before I believe, however some users reported the user interface freezing after a batch of 200 completing, I think they were just being impatient, we will likely increase the batch size slider upper bound to a more sensible limit that won't freeze the gallery, and add a second option with a text box to enter a number but that will then disable gallery output, could probably add a progress bar or something instead though

**Comment by altryne** [comment]
It's hard for us to access the usage of features as we have no analytics (@hlky I'll add some basic ones)

@JamesIV4 @Aishou you guys can override the limits now with dynamic params in the prompt. This was announced here : 
https://github.com/hlky/stable-diffusion-webui/discussions/422

so if you type numer:100 in your prompt it will allow you to go out of the boundaries of the defaults easily without changing code

**Comment by altryne** [comment]
The issue with a LOT by default was twofold: 
1) was really hard on smaller screens to nudge the slider from 4 to 5 for example, and many folks don't do HUGE batches
2) The batch size and batch count were confusing and folks tried and got a CUDA error cause.. it's heavy, and continued opening tickets 
3) Gradio UI doesn't give an option to add arbitrary values to sliders (it's on open issue of theirs)

**Comment by hlky** [comment]
We don't need analytics they wont tell us anything issue reports can't. Don't add any analytics.

**Comment by hlky** [comment]
@JamesIV4 batch count was increased to 50

**Comment by JamesIV4** [comment]
50 is great, and nice to know about the override, very useful 👍
